src/main/webapp/user-settings/result-form-select.js

```diff
@@ -15,10 +15,23 @@ const resultForms = gql`
         title
       }
     }
+    user {
```

Comment:
I think this data is already passed to this component from https://github.com/connexta/revelio/blob/master/src/main/webapp/user-settings/user-settings.js#L209
https://github.com/connexta/revelio/blob/master/src/main/webapp/user-settings/user-settings.js#L170

Reply:
I wasn't sure the best way to handle this. `result-form-selector` is used by other components than just `user-settings`, so I think it needs to have its own reference to user preferences, unless you have a better idea for how to approach this?